logger.js

```diff
@@ -62,6 +78,7 @@ function pinoLogger (opts, stream) {
     res.on('error', onResFinished)
     // it's possible that browser aborts connection, or http-server because of timeout
     req.on('aborted', onReqAborted)
+    req.socket.on('timeout', onReqTimeout.bind(req))
```

Comment:
I think `'timeout'`  is not emitted if `setTimeout()`  is not called on the request or response.
Maybe is what we want, would you mind setting `setTimeout` explicitly?

`bind` is extremely slow on node v4 and v6, it should not be used.